scripts/us_census/pep/monthly_population_estimate/download.py

```diff
@@ -0,0 +1,226 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" A Script to download, perform some basic transformations to
+    USA Census PEP monthly population data from the URLS in
+    provided json file and save it as an xlsx file.
+"""
+
+import os
+import json
+import pandas as pd
+import numpy as np
+from absl import app
+from absl import flags
+
+_FLAGS = flags.FLAGS
+_URLS_JSON_PATH = os.path.dirname(os.path.abspath(__file__)) \
+                    + os.sep +"file_urls.json"
```

Comment:
prefer to use os.path.join? 

Its used more in the repo and is generally more readable:
https://stackoverflow.com/questions/16789714/differences-between-use-of-os-path-join-and-os-sep-concatenation

Reply:
Added os.path.join where ever possible